syntax/success_test.go

```diff
@@ -0,0 +1,234 @@
+package syntax
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func unhex(encoded string) []byte {
+	decoded, err := hex.DecodeString(encoded)
+	if err != nil {
+		panic(err)
+	}
+	return decoded
+}
+
+func buffer(size int) []byte {
+	return bytes.Repeat([]byte{0xA0}, size)
+}
+
+func hexBuffer(size int) string {
+	return strings.Repeat("A0", size)
+}
+
+type CrypticString string
+
+var (
+	crypticStringMarshalCalls   = 0
+	crypticStringUnmarshalCalls = 0
```

Comment:
Do you check these?

Reply:
No, and I don't think there's much value to checking, and it's hard to do in the current framework.  So I've deleted them.